src/critical-css.js

```diff
@@ -0,0 +1,20 @@
+import { existsSync, lstatSync, readFileSync } from 'fs';
```

Comment:
Right now the module does too little to stand on its own (I'd move the code to server.jsx). Maybe we can cache the css content (if found) for 5 minutes? Then it will be ok to be standalone.

Reply:
I added the cache feature.